django-stubs/contrib/gis/gdal/layer.pyi

```diff
@@ -41,7 +41,7 @@ class Layer(GDALBase):
     spatial_filter: OGRGeometry | None
     def get_fields(self, field_name: str) -> list[Any]: ...
     @overload
-    def get_geoms(self, geos: Literal[True] = ...) -> OGRGeometry: ...
+    def get_geoms(self, geos: Literal[True] = ...) -> list[GEOSGeometry]: ...
     @overload
-    def get_geoms(self, geos: Literal[False]) -> GEOSGeometry: ...
+    def get_geoms(self, geos: Literal[False]) -> list[OGRGeometry]: ...
```

Comment:
Looks like the default for `geos=` is False, so the `= ...` should be moved to the second overload?

Doc: https://docs.djangoproject.com/en/4.1/ref/contrib/gis/gdal/#django.contrib.gis.gdal.Layer.get_geoms

Reply:
You are right. Thanks!